src/gsy_e/models/area/__init__.py

```diff
@@ -252,6 +252,19 @@ def get_results_dict(self):
         return {}
 
 
+class CoefficientAreaException(Exception):
+    """Exception that is raised when serializing an area."""
+
+
+def assert_coefficient_area_settings(
+        children: Optional[List], setting: Optional[float], setting_name: str) -> float:
+    """Check if coefficient area that is not an asset provided SCM setting."""
+    if children and len(children) > 0:
```

Comment:
```suggestion
    if children:
```
If a list is empty its truth value is `False`.

Reply:
It could also be set to None.

Reply:
`None` if a false-y value too, so `if children` is `False` if `children == None`.

Reply:
Ah yes, and if it is an empty list, if will also not go into the if block. My bad! 
Applied here: [7ba425d](https://github.com/gridsingularity/gsy-e/pull/1544/commits/7ba425d05ff0cd3cb2fb9c35f02d8515af156048)